### ISSUE TITLE
[ACCESSIBILITY] Unclear labeling on Daily Wellness rings (screen reader user unable to comprehend content)

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#2023]
**Describe the accessibility issue**
As a visually disabled user, I'm navigating the Daily wellness rings area of the app and all I can hear is " 0%, 0%, 0%" and I have no idea what these percentages or numbers are trying to describe. This needs proper labeling and context for the non-visual user. 

**To Reproduce
Steps to reproduce the behavior:**
USING A SCREEN READER, gesture through app to the Daily Wellness Rings area
Swipe through to hear

**Expected behavior**
A clear and concise description of what sighted user is seeing/reading is read to screen reader user. 

**Actual behavior**
User is hearing  " 0%, 0%, 0%" and I have no idea what these percentages or numbers are trying to describe.
(This also happens with rings area on Favorites screen. )

Screenshots
![Screenshot_20220715-151724](https://user-images.githubusercontent.com/82892873/179310455-65d030e3-d238-462c-bed3-ce9809ec3f31.png)
![Screenshot_20220715-151647](https://user-images.githubusercontent.com/82892873/179310463-6d9a1375-655d-43c2-9473-5a8bcc09d5b4.png)

Smartphone (please complete the following information):
Google Pixel 4a Android 12

Additional context

**Comment by TodorBachvarov** [comment]
Fixed in version 4.1.3

**Comment by acovher2** [comment]
Verified as fixed in 4.1.19
